Cookbook/Proposals/Template_Proposal_Meeting_Invitation.md

```diff
@@ -1,27 +1,32 @@
 Hi all,
 
-This meeting is to discuss the "<Proposal Title>" proposal authored by <Proposal's Author>.
+This meeting is to discuss the "<#Proposal Title#>" proposal authored by <#Proposal's Author#>.
 
-1.	PR here <Link to the Proposal's PR>.
-2.	Actual proposal here <Link to the Proposal's .md file>.
+1.	PR here <#Link to the Proposal's PR#>.
+2.	Actual proposal here <#Link to the Proposal's .md file#>.
 
 Meeting Etiquette:
 
 1.	**Please fully read the proposal.**
-  a.	If you haven’t read the proposal, or don’t care for the outcome, please don’t show up for the meeting.
+
+   * If you haven’t read the proposal, or don’t care for the outcome, please don’t show up for the meeting.
```

Comment:
I feel like this is a bit too aggressive. Would rephrase to something like

```suggestion
   * If you are not interested in participating in the proposal discussion you are free to skip the meeting. If you are, please make sure to have read the document and its discussion carefully before the meeting starts, as the proposal itself will not be explained again.
```

Reply:
@diegopetrucci I agree about changing the wording, but I read your suggestion as "[...] you are free to skip the meeting. If you are **[skipping the meeting]**, please make sure to have read [...]" which was confusing 😅 open for suggestions on how to improve that

Reply:
@diegopetrucci how about:
```suggestion
   * Please make sure to have read the document and its discussion carefully before the meeting starts, as the proposal itself will not be explained again.
   * If you are not interested in participating in the proposal discussion, you are free to skip the meeting.
```

Reply:
go ahead with yours 🚀